js/constants/constants.js

```diff
@@ -0,0 +1,8 @@
+var keyMirror = require('react/lib/keyMirror');
+
+module.exports = {
+  ActionTypes: keyMirror({
+    NEW_MESSAGE: null,
+    RENAME_LOCAL_USER: null
+  })
```

Comment:
This pattern seems okay after using it for something. But one shitty thing is that ActionTypes.MISPELLED_ACTION will === ActionTypes.OTHER_SHITTY_MISPELLLED_ACTION will be true because both are null. Is there anything like ruby's fetch that will throw an error if you try to access a missing object attribute?